server/checks/Autobranch.js

```diff
@@ -0,0 +1,62 @@
+import { logger } from '../../common/debug'
+
+const info = logger('autobranch', 'info')
+const error = logger('autobranch', 'error')
+
+function safeArray(arr) {
+  if (!Array.isArray(arr) || arr.length === 0) {
+    return ''
+  }
+  return arr.map(safeString).join('-')
+}
+
+function safeString(str) {
+  return str
+          // drop everything from title that is not ascii
+          .replace(/[^a-zA-Z0-9\s]+/gu, '')
```

Comment:
I don't think it's necessary to drop non-ascii letters. Both Github and Git support them just fine. Also it would be simpler to just do:

```javascript
str
  .trim()               // get rid of any leading or trailing whitespace
  .replace(/\s+/g, '-') // replace " " or "  " or "   " etc. with "-"
  .toLowerCase()
```